code/game/atom/atom_act.dm

```diff
@@ -83,26 +83,26 @@
  * * def_zone - zone hit
  * * piercing_hit - is this hit piercing or normal?
  */
-/atom/proc/bullet_act(obj/projectile/hitting_projectile, def_zone, piercing_hit = FALSE)
+/atom/proc/bullet_act(obj/projectile/proj, def_zone, piercing_hit = FALSE, blocked = check_projectile_armor(def_zone, proj))
```

Comment:
please never ever do this just be explicit in the proc you want me dead

Reply:
I think it's based